prody/dynamics/analysis.py

```diff
@@ -747,4 +747,4 @@ def calcScipionScore(modes):
 
     return score
 
-calcHemnmaScore = calcScipionScore
\ No newline at end of file
+calcHemnmaScore = calcScipionScore
```

Comment:
what happened here?

Reply:
*GitHub* seems to have added an empty line at the end of file. I created this PR directly via the editor on *GitHub*.

Reply:
ok, I think that's reasonable